app/build.gradle

```diff
@@ -5,12 +5,12 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-kapt'
 
 android {
-  compileSdkVersion 33
-  buildToolsVersion "30.0.2"
+  compileSdkVersion 34
+  buildToolsVersion "30.0.0"
```

Comment:
Acking that these aren't correct, but I don't care to update them since we're disabling Gradle in the base PR, anyway (#5629).

Reply:
Resolving since there's no action to take here--I'm just documenting this for posterity.